File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.cpp
class Solution {
public:
    double average(vector<int>& salary) {
        int mn=*min_element(salary.begin(),salary.end());
        int mx=*max_element(salary.begin(),salary.end());
        double res=0.0;
        for(int i:salary)res+=i;
        
        return (res-mn-mx)/(salary.size()-2);
        
    }
};

File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.java
class Solution {
    public double average(int[] salary) {
        int mn=Integer.MAX_VALUE,mx=Integer.MIN_VALUE;
        double res=0.0;
        for(int i:salary){
            if(i<mn)mn=i;
            if(i>mx)mx=i;
            res+=i;
        }
        return (res-mn-mx)/(salary.length-2);
    }
}